src/microbe_stage/MicrobeAI.cs

```diff
@@ -418,18 +183,12 @@ private Microbe GetNearestPreyItem(List<Microbe> allMicrobes)
 
         foreach (var otherMicrobe in allMicrobes)
         {
-            if (otherMicrobe == microbe || otherMicrobe == previousPrey)
-                continue;
-
-            if (otherMicrobe.Species != microbe.Species && !otherMicrobe.Dead)
+            if (!otherMicrobe.Dead)
             {
-                if ((SpeciesAggression == Constants.MAX_SPECIES_AGRESSION) ||
-                    ((((microbe.AgentVacuoleCount + microbe.EngulfSize) * 1.0f) *
-                            (SpeciesAggression / Constants.AGRESSION_DIVISOR)) >
-                        (otherMicrobe.EngulfSize * 1.0f)))
+                if (DistanceFromMe(otherMicrobe.Translation) <
```

Comment:
This will make them just always grab the closet one to them, this isn't always ideal, which is why the previous system allowed the ai to chase a single specific microbe for some time  (they will just sit their with decision paralysis if there are tons of microbes around them at similar Distances with this. because some will get closer's then half a second later something else will be closer and they will switch and so on. The pegged prey stuff was not redundant. Did you actually test the predation behavior?

Reply:
In testing I only saw a half-dozen cases of predation (cases where a microbe SHOULD hunt are pretty rare in my experience, and this AI is much less suicidality aggressive in response to that), but so far I never saw a predator get stuck between two prey items. With most cells moving most of the time, I would expect that scenario to be a very rare edge case, and one that would last seconds at most. I can do some more playtesting to see if I can get that behavior to occur.

Reply:
> In testing I only saw a half-dozen cases of predation (cases where a microbe SHOULD hunt are pretty rare in my experience, and this AI is much less suicidality aggressive in response to that), but so far I never saw a predator get stuck between two prey items. With most cells moving most of the time, I would expect that scenario to be a very rare edge case, and one that would last seconds at most. I can do some more playtesting to see if I can get that behavior to occur.

buckly saw the indesisive ness. You may not notice it but the ai is entering the predation and choosing targets over and over again as I suspected every few seconds. You will have to bring pack the pegging.

Reply:
They should also peg chunks because  as they swim  towards a chunk they might get closer to other chunks which are moving around because of the currents/getting pushed.

Reply:
I ran into these same issues when i first implemented the ai which is why I had to do these things. So its okay. I didnt thoroughly comment things to explain why there were there but most of it had a purpose.

Reply:
There's also the issue of it being fun. Its not fun to not get chased around sometimes. So the ai should definitely not just target the closest one.

Reply:
Writing a good ai isnt just about the ai living longest its about making a fun game